includes/functions/updates/llms-functions-updates-600.php

```diff
@@ -81,15 +81,21 @@ function migrate_award_templates() {
 		)
 	);
 
-	$legacy_option_added = false;
+	$legacy_option_added['achievement'] = false;
+	$legacy_option_added['certificate'] = false;
```

Comment:
```suggestion
	$legacy_option_added = array(
		'achievement' => false,
		'certificate' => false,
	);
```
is it any better or not, dunno...

Reply:
I first did it that way. It allows us to easily add more elements. Except, we aren't going to add any more engagement types in the 6.0 update functions and it uses 4 lines instead of 2.

Reply:
I don't know, for some reason I feel `=array(a,b,c)` resembles a variable declaration.